docs/registry.md

```diff
@@ -0,0 +1,341 @@
+----------|------------------------------------
+page      | the page of the list
+limit     | the maximum number of applications to show
+filter[]  | a filter to apply on fields of the applicaiton
+order     | order to apply to the list
+
+#### Request
+
+```http
+GET /?filter[category]=cozy&page=0&limit=30
+```
+
+#### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+[
+    {
+        "name": "drive",
+        "type": "webapp",
+        "editor": "cozy",
+        "category": "files",
+        "description": "The drive application",
+        "versions": ["v3.1.1","v3.1.2"],
+        "versionsDev": ["v3.1.1-7ed536cc0a6c2a7daa7e961bdfd5ef5fe763a442"],
+        "repository": "https://github.com/cozy/cozy-drive",
+        "license": "BSD"
+    },
+    {
+        // ...
+    }
+]
+```
+
+### GET /:app
+
+Get an application object by name.
+
+#### Request
+
+```http
+GET /drive
+```
+
+#### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+    "name": "drive",
+    "type": "webapp",
+    "editor": "cozy",
+    "description": "The drive application",
+    "versions": ["v3.1.1","v3.1.2"],
+    "versionsDev": ["v3.1.1","v3.1.1-7ed536cc0a6c2a7daa7e961bdfd5ef5fe763a442", "v3.1.2"],
+    "repository": "https://github.com/cozy/cozy-drive",
+    "license": "BSD"
+}
+```
+
+### GET /:channel/:app/:version
+
+Get an application version on the specified channel (stable or dev).
+
+#### Request
+
+```http
+GET /stable/drive/v3.1.1
+```
+
+#### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+    "version": "v3.1.1",
+    "url": "http://.../v3.1.1",
+    "sha256": "466aa0815926fdbf33fda523af2b9bf34520906ffbb9bf512ddf20df2992a46f",
+    "channel": "stable",
+    "size": "1000",
+    "created_at": "2017-07-05T07:54:40.982Z",
+    "description": "Description of the 3.1.1 version of drive",
+    "permissions": { },
+    "locales": { }
+}
+```
+
+### GET /:channel/:app/:latest
+
+Get the latest version available on the specified channel.
+
+#### Request
+
+```http
+GET /dev/drive/latest
+```
+
+#### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+    "version": "v3.1.1",
+    "url": "http://.../v3.1.1",
+    "sha256": "466aa0815926fdbf33fda523af2b9bf34520906ffbb9bf512ddf20df2992a46f",
+    "channel": "stable",
+    "size": "1000",
+    "created_at": "2017-07-05T07:54:40.982Z",
+    "description": "Description of the 3.1.1 version of drive",
+    "permissions": { },
+    "locales": { }
+}
+```
+
+## Attaching a cozy-stack to a registry or a list of registries
+
+In the configuration file of a stack, a `registries` namespace is added. This namespace can contain a list of url for the different registries attached to the stack.
+
+The stack itself implements the querying API of a registry. When querying this API, to ask for an application, the stack uses this hierarchy of registries to proxy or redirect the user.
+
```

Comment:
In that case, the SERIOUSPARTNER should have a context without the default registry, no?